bitcoin/src/blockdata/transaction.rs

```diff
@@ -778,12 +778,14 @@ impl Transaction {
             size += 2; // 1 byte for the marker and 1 for the flag.
         }
 
+	let use_segwit = self.use_segwit_serialization();
```

Comment:
You should move it above the if and use it in if too.

Reply:
Holy moley, I am constantly surprised/disapointed by my tunnel vision.

Reply:
Tunnel vision can help with focus sometimes :)